tutorial/markdown/nodejs/quickstart-nextjs-capella/quickstart-nextjs-capella.md

```diff
@@ -0,0 +1,804 @@
+
+- Ensure that the Nodejs version is [compatible](https://docs.couchbase.com/nodejs-sdk/current/project-docs/compatibility.html) with the Couchbase SDK.
+- Loading Travel Sample Bucket
+  If travel-sample is not loaded in your Capella cluster, you can load it by following the instructions for your Capella Cluster:
+
+  - [Load travel-sample bucket in Couchbase Capella](https://docs.couchbase.com/cloud/clusters/data-service/import-data-documents.html#import-sample-data)
+
+> Note that this tutorial is designed to work with Next.js 14. Please make sure you have Next.js 14 installed before proceeding.
+
+## App Setup
+
+### Cloning Repo
+
+```shell
+git clone https://github.com/couchbase-examples/nextjs-capella-quickstart.git
+```
+
+### Install Dependencies
+
+Any dependencies should be installed through the node package manager (npm). Navigate to the root directory and run the following command:
+
+```shell
+npm install
+```
+
+### Setup Database Configuration
+
+To know more about connecting to your Capella cluster, please follow the [instructions](https://docs.couchbase.com/cloud/get-started/connect.html).
+
+Specifically, you need to do the following:
+
+- Create the [database credentials](https://docs.couchbase.com/cloud/clusters/manage-database-users.html) to access the travel-sample bucket (Read and Write) used in the application.
+- [Allow access](https://docs.couchbase.com/cloud/clusters/allow-ip-address.html) to the Cluster from the IP on which the application is running.
+
+All configuration for communication with the database is fetched from the environment variables. We have provided a convenience feature in this quickstart to read the environment variables from a local file, `.env` in the source folder.
+
+Create a copy of `.env.example` file & rename it to `.env.local` & add the values for the Couchbase cluster.
+
+```sh
+DB_CONN_STR=<connection_string>
+DB_USERNAME=<user_with_read_write_permission_to_travel-sample_bucket>
+DB_PASSWORD=<password_for_user>
+```
+
+> Note: The connection string expects the `couchbases://` or `couchbase://` part.
+
+## Running the Application
+
+### Directly on Local Machine
+
+At this point, we have installed the dependencies, loaded the travel-sample data and configured the application with the credentials. The application is now ready and you can run it.
+
+```sh
+npm run dev
+```
+
+The application will start on port 3000 of your local machine (http://localhost:3000).
+The interactive Swagger documentation of the API can be found at http://localhost:3000/api-doc.
+
+#### Using Docker
+
+If you prefer to run this quick start using Docker, we have provided the Dockerfile which you can use to build the image and run the API as a container.
+
+- Build the Docker image
+
+```sh
+docker build -t docker build -t nextjs-capella-quickstart . -f Dockerfile.dev
```

Comment:
Is this command tested? I doubt that it works.

Reply:
oops. removed the duplicate words